scripts/istio-utils.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+# A set of utility functions for working with Istio
+# Calls to kubectl expects KUBECONFIG environment variable to be set correctly.
+
+
+script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+root_dir=$(dirname "${script_dir}")
+output_dir="${root_dir}"/_output
```

Comment:
there are several instances of this hardcoded instead of referencing this variable